metadata-ingestion/src/datahub/ingestion/source/dbt.py

```diff
@@ -125,6 +129,65 @@ class DBTStatefulIngestionConfig(StatefulIngestionConfig):
     remove_stale_metadata: bool = True
 
 
+class DbtCheckpointState(CheckpointStateBase):
```

Comment:
Refactor this out to metadata-ingestion/src/datahub/ingestion/source/state/dbt_state.py per convention (@ralanka)